client/components/FirebaseImg.tsx

```diff
@@ -28,17 +31,25 @@ const FirebaseImg = (props) => {
       }, []);
   
     return (
-        <Image           
-            style={{
-                height: imgProps.height,
-                width: imgProps.width,
-                borderRadius: imgProps.borderRadius,
-                marginRight: imgProps.marginRight,
-                aspectRatio: imgProps.aspectRatio,
-            }} 
-            source={{uri: imageUrl}} 
-        />
+        <Image
```

Comment:
As discussed, let's keep the borderRadius, marginRight and aspectRatio. 

Also could you switch height 85 and width 60 so the default can be landscape?

Reply:
Yep, I'll have it in the next one!